public/content/en/vibed/deploy-on-heroku.md

```diff
@@ -0,0 +1,188 @@
+# Deploy on Heroku
+
+### Pre-requirements
+
+- Heroku [account]((https://signup.heroku.com/login))
+- [Git](https://git-scm.com/) installed
+- Your application should compile without error using dmd (`dub build --build=release`). 
+
+### 1 : Setup the app
+
+The first thing to do before deploying is binding the app's port to heroku's one. 
+Heroku sets the `PORT` variable that you are supposed to bind, and listens on port 80 by default.
+
+```d
+shared static this() {
+  // ...
+  auto settings = new HTTPServerSettings;
+  // Provide a default port in case of the $PORT variable isn't set.  
+  settings.port = environment.get("PORT", "8080").to!ushort;
+  listenHTTP(settings, router);
+}
+```
+
+Also create a `Procfile`, which is a text file in the root directory of the application, which explicitly declare what command 
```

Comment:
Try to avoid "Also" - it's a rather colloquial, better use "Furthermore", "Moreover", "Additionally", ...